docs/examples/applied_heatwave.py

```diff
@@ -33,11 +35,19 @@
     variable_mapping=inputs.HRES_metadata_variable_mapping,
 )
 
+# Load the climatology for DurationME
+climatology = xr.open_zarr(
+    "gs://extremeweatherbench/datasets/surface_air_temperature_1990_2019_climatology.zarr",  # noqa: E501
+    storage_options={"anon": True},
+    chunks="auto",
+)
+climatology = climatology["2m_temperature"].sel(quantile=0.85)
```

Comment:
Naming her is a tad confusing since you pass it to a kwarg "criteria" later on. Maybe rename this something like "temp_threshold" for clarity.